packages/@lwc/ssr-runtime/src/lightning-element.ts

```diff
@@ -13,10 +13,12 @@
 // and be located before import statements.
 // /// <reference lib="dom" />
 
-import { htmlPropertyToAttribute } from '@lwc/shared';
+import { defineProperties } from '@lwc/shared';
+
 import { ClassList } from './class-list';
 import { Attributes } from './types';
 import { mutationTracker } from './mutation-tracker';
+import { reflectAttrToProp, descriptors as reflectionDescriptors } from './reflection';
```

Comment:
Circular imports 😕

Reply:
Are you referring to how `lightning-element.ts` imports from `reflection.ts` and vice versa?

Reply:
Yeah. Not too big a deal, and using `import type` in `reflection.ts` will at least resolve it at runtime.